lib/rbac/filterer.rb

```diff
@@ -417,17 +402,33 @@ def scope_to_tenant(scope, user, miq_group)
       tenant_id_clause ? scope.where(tenant_id_clause) : scope
     end
 
+    ##
+    # Main scoping method
+    #
     def scope_targets(klass, scope, rbac_filters, user, miq_group)
+      # Results are scoped by tenant if the TenancyMixin is included in the class,
+      # with a few manual exceptions (User, Tenant). Note that the classes in
+      # TENANT_ACCESS_STRATEGY are a consolidated list of them.
       if klass.respond_to?(:scope_by_tenant?) && klass.scope_by_tenant?
```

Comment:
I'm not sure what you mean here ^ But I also like the idea of an RBAC lookup table rather than classes responding to certain methods (I just think we can make more of a registry as opposed to a hardcoded table). 

In any case, this logic isn't changing in this PR, so let's move this conversation elsewhere. I'm not going to change the implementation right now, but I am welcome to changing the comment if someone is confused by it.